neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/BoltNeo4jStatementIT.java

```diff
@@ -305,4 +305,36 @@ public static void tearDown(){
 		statement.close();
 	}
 
+	@Test public void testCallProcedure() throws SQLException {
```

Comment:
it'd be good to add a query or a procedure that returns nothing?
maybe `UNWIND [] AS nope RETURN nope`?

Reply:
forget the above suggestion (no returned records does not mean `EmptyResult` operator)
a query such as `MATCH (n:FooBar) SET n.baz = true` or `CREATE (blah)` should be tested as well

Reply:
added